backend/account/views.py

```diff
@@ -37,9 +37,10 @@ def profile(request, user_pk):
     """
     prof = get_object_or_404(Profile, pk=user_pk)
     if request.method == 'GET':
-        if not request.user.is_authenticated:
-            return HttpResponse(status=401)
+        # if not request.user.is_authenticated:
+        #    return HttpResponse(status=401)
         dict_profile = model_to_dict(prof)
+        dict_profile['username'] = prof.__str__()
```

Comment:
use `str(prof)` instead?

Reply:
Thanks for your feedback! I will fix it.